### PR TITLE
Leaderboard

### PR DESCRIPTION
**Pull request by AmaliaGM**
## For this project

-   Create an npm project with webpack. 
-   Write plain HTML markup with minimum styling (just to make the layout), because you will implement styles in the following steps. Use the wireframe as a guide:
-   Don't make API calls.
-   Use ES6 modules, with import and export.

### CODE REVIEW

File: addElements.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+         score = JSON.stringify(inputScore, null, 2);
+    }
+        document.getElementsByClassName('nameInput').value;
+        document.getElementsByClassName('scoreInput').values;
+ */
+  /* for (let i = 0; i < scoreList.length; i += 1) {
+          console.log(scoreList[i]);
+        }
+ */
+}
```
Comment (keneogu) [inline addElements.js]: Remove the commented out code.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+         score = JSON.stringify(inputScore, null, 2);
+    }
+        document.getElementsByClassName('nameInput').value;
+        document.getElementsByClassName('scoreInput').values;
+ */
+  /* for (let i = 0; i < scoreList.length; i += 1) {
+          console.log(scoreList[i]);
+        }
+ */
+}
```
Comment (keneogu) [inline src/index.js]: Remove the commented codes

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @AmaliaGM,

Good job so far!

- All linters are passing
- You are following GitHub flow
- Good commit history

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] Remove the live link on your readme file since it gives error when clicked and also add how to run the code.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @AmaliaGM, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AmaliaGM** [comment]
@Deepakdanger Thanks so much! Happy coding!

File: src/index.js (hunk 2 above)
Comment (AmaliaGM) [reply src/index.js]: @keneogu Thanks for your suggestions!

**Pull request merged**